gpkit/nomial_data.py

```diff
@@ -54,6 +45,24 @@ def __hash__(self):
             self._hashvalue = hash(tuple(zip(self.exps, self.cs)))
         return self._hashvalue
 
+    @classmethod
+    def fromnomials(cls, nomials):
+        """Construct a NomialData from an iterable of Signomial objects"""
+        nd = cls()  # use pass-through version of __init__
```

Comment:
Yeah. Any reason to use the pass through rather than just modifying the object in `fromnomials`?